CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+> ```
+
+To build **Lawnchair**, select the `lawnWithQuickstepDebug` build type.
+Should you face errors relating to the `iconloaderlib` and `searchuilib` projects,
+run `git submodule update --init --recursive`.
+
+Here are some contribution tips to help you get started:
+
+-   Always make sure that you're up-to-date with **Lawnchair** by setting your branch to `14-dev` or newer;
+-   Make sure your code is logical and well-formatted. If using Kotlin, see [“Coding conventions” in the Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
+-   [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum.
+
+#### Additional documentation
+
+-   Lawnchair Quickstep Compat Lib ([README.md](compatLib/README.md))
+-   Lawnchair Preferences Components ([README.md](lawnchair/src/app/lawnchair/ui/preferences/components/README.md))
+-   SystemUI ViewCapture ([README.md](systemUIViewCapture/README.md))
+-   SystemUI Common ([README.md](systemUICommon/README.md))
+-   Prebuilt Library ([README.md](prebuilts/libs/README.md))
+
+#### Versioning scheme
+
+As of [#4361](https://github.com/LawnchairLauncher/lawnchair/pull/4361), 
+the **Lawnchair** version code is separated by four parts:
+
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="docs/version-dark.svg" width="98%">
+        <img alt="" src="docs/version-light.svg" width="98%">
+        <!-- Direct the accessibility reader to read the point below --->
+    </picture>
+</p>
+
+1. Android's major version
+2. Android's minor version
+3. Lawnchair's development status
+4. Lawnchair's development version
+
+##### Android's major & minor versions
+
+These makes up the first two parts of the version code, 
+Android 11 will be `11_00_XX_XX` while Android 12.1 will be `12_01_XX_XX`.
+
+##### Development status & version
+
+Depending on which status the **Lawnchair** is on, 
+it will have an impact on the third and fourth parts of the version code. 
+For example, alpha 4 for the **Lawnchair** will be `XX_XX_01_04`. 
+
+Below is a table explaining the third part (release type):
+
+| Status            | Stage |
+| ----------------- | ----- |
+| Alpha             | 01    |
+| Beta              | 02    |
+| Release Candidate | 03    |
+| Release           | 04    |
+
+## Quick links
+
+-   [News](https://t.me/lawnchairci)
+-   [Lawnchair on Twitter](https://twitter.com/lawnchairapp)
+-   [Website](https://lawnchair.app)
+-   [_XDA_ thread](https://forum.xda-developers.com/t/lawnchair-customizable-pixel-launcher.3627137/)
+
+You can view all our links in the [Lawnchair Wiki](https://github.com/LawnchairLauncher/lawnchair/wiki).
```

Comment:
Probably an oversight; please remove this since it isn't relating to contributing to Lawnchair.

```suggestion

```